RELEASING.md

```diff
@@ -14,5 +14,22 @@ kpt-functions-catalog repo.
 1. Click `Publish release` button.
 1. Send an announcement email in the [kpt users google group].
 
+## Updating function docs
+
+After creating a release, the docs for the function should be updated to reflect
+the latest patch version. A script has been created to automate this process.
+
+1. Run the script:
+```shell
+# Check out latest version of the make target from master
+git fetch upstream && git checkout upstream/master
+# Run the make target
+# e.g. RELEASE_BRANCH=set-namespace/v0.3 make update-function-docs
+RELEASE_BRANCH=<RELEASE_BRANCH> make update-function-docs
+```
+2. The script will generate a new commit in your local repository which updates
```

Comment:
nit: Use `1.` here and it will be shown in the list form. See example above

Reply:
Done